app/views/apply/_variant-choice-with-sign-in.njk

```diff
@@ -0,0 +1,33 @@
+<p class="govuk-body">{{ serviceName }} is a new GOV.UK service that will eventually replace UCAS, but it is not suitable for everyone yet.</p>
```

Comment:
> Apply for teacher training is a new GOV.UK service...

Just my 2p: this may read strangely if you don’t know that ‘Apply for teacher training’ is a service name, as opposed to an imperative to apply.

Reply:
Yup, there’s a fine line between you can I use _Apply for teacher training_ (the service), and can I _apply for teacher training_. This is where naming services to describe the activity falls apart, even more so when there are two services that do the same thing.

We’ll be looking at this more closely next week when @emmajhf is back in.